docs/apis/webadmin.md

```
@@ -364,3 +364,47 @@ Revokes administrator rights of user `user1@linagora.com` for domain `linagora.c
+---
+
+### Running the resource rights reposition task
+
+This task allows re-applying or correcting CalDAV resource rights based on the current resources administrators stored
+in the database.
+
+```
+POST /resources/tasks?task=repositionWriteRights
+```
+
+**Status codes**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)